src/libs/Navigation/linkTo/index.ts

```diff
@@ -184,7 +184,13 @@ export default function linkTo(navigation: NavigationContainerRef<RootStackParam
         }
     }
 
-    if (action && 'payload' in action && action.payload && 'name' in action.payload && isSideModalNavigator(action.payload.name)) {
+    if (
+        action &&
+        'payload' in action &&
+        action.payload &&
+        'name' in action.payload &&
+        (isSideModalNavigator(action.payload.name) || action.payload.name === NAVIGATORS.FULL_SCREEN_NAVIGATOR)
+    ) {
```

Comment:
These navigation changes were consulted with me. We need to `PUSH` screens in FullScreenNavigator instead of `NAVIGATE` to ensure animations and history work properly